DataFormats/TotemRPReco/interface/TotemRPRecHit.h

```diff
@@ -0,0 +1,55 @@
+/****************************************************************************
+*
+* This is a part of TOTEM offline software.
+* Authors:
+*   Hubert Niewiadomski
+*   Jan Kašpar (jan.kaspar@gmail.com)
+*
+****************************************************************************/
+
+#ifndef DataFormats_TotemRPReco_TotemTotemRPRecHit
+#define DataFormats_TotemRPReco_TotemTotemRPRecHit
+
+/**
+ *\brief Reconstructed hit in TOTEM RP.
+ *
+ * Basically a cluster (TotemRPCluster), the position of which has been converted into actual geometry (in mm).
+ **/
+class TotemRPRecHit
+{
+ public:
+  TotemRPRecHit(unsigned int det_id, double position, double sigma) : det_id_(det_id), 
+    position_(position), sigma_(sigma) {}
+  TotemRPRecHit() : det_id_(0), position_(0), sigma_(0) {}
+
+  inline void Position(double position) {position_=position;}
```

Comment:
@grzanka 
It would be the best if you can address the issues similar to what was already brought up in the earlier review.
We have ~30 PRs in the pipeline now. 
So, full set of comments will come later this week.